### PR TITLE
Exclude some links in check-links.yml

### PR DESCRIPTION
**Pull request by core-man**
Exclude some valid links in `check-links.yml` reported in #211.

### CODE REVIEW

File: .github/workflows/check-links.yml [hunk 1]
```diff
@@ -31,6 +31,15 @@ jobs:
         # 429: Too many requests
         args: >
           --accept 429
+          --exclude "^http://www.seis.sc.edu/software/distaz/distaz.c/n$"
+          --exclude "^http://www.seis.sc.edu/software/distaz/distaz.c/n/n#$"
+          --exclude "^https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb/n$"
+          --exclude "^https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB/n#$"
```
Comment (seisman) [inline .github/workflows/check-links.yml:37]: lychee sometimes doing bad with parsing URLs. Perhaps we should exclude all URLs ending with `/n` and `/n#`?

Comment (core-man) [reply .github/workflows/check-links.yml:37]: `"*/n$"`, `"*/n#$"`?

Comment (core-man) [reply .github/workflows/check-links.yml]: done in d234b2c.

Comment (seisman) [reply .github/workflows/check-links.yml]: It may work or not. You can enable the workflow in this PR to check how it works.

Comment (core-man) [reply .github/workflows/check-links.yml]: Good. Works (#214)

File: .github/workflows/check-links.yml [hunk 2]
```diff
@@ -31,6 +31,13 @@ jobs:
         # 429: Too many requests
         args: >
           --accept 429
+          --exclude "*/n$"
+          --exclude "*/n#$"
```
Comment (seisman) [inline .github/workflows/check-links.yml:35]: Use regex, not glob pattern.
```suggestion
          --exclude ".*/n$"
          --exclude ".*/n#$"
```

File: .github/workflows/check-links.yml (hunk 1 above)
Comment (seisman) [reply .github/workflows/check-links.yml]: No, it doesn't work.

**Comment by seisman** [comment]
Please enable it in PR to test it again.

File: .github/workflows/check-links.yml [hunk 3]
```diff
@@ -3,7 +3,7 @@ name: Check Links
 
 on:
   # Uncomment the 'pull_request' line below to trigger the workflow in PR
-  # pull_request:
+  pull_request:
```
Comment (seisman) [inline .github/workflows/check-links.yml:6]:
```suggestion
  # pull_request:
```

**Review by seisman — APPROVED** [review]

**Pull request merged**